tests/distributed_grids/1d_grid.expect=build.output

```diff
@@ -0,0 +1,106 @@
+
+DEAL:2d::hyper_cube
```

Comment:
Is this actual output, or do you just need to provide *any* kind of file for the test to work? If the latter, just use an empty file.

Reply:
it looks like we need any file, including an empty one for `test.expected=build.output` to pass.